protocols/022548-2/022548-2.ot2.apiv2.py

```diff
@@ -0,0 +1,820 @@
+
+    # Todo: If tuberack: Error check that all tubes are 15 mL types
+
+    # Define labware and slots
+    sample_tuberack_loader = \
+        ("nest_32_tuberack_8x15ml_8x15ml_8x15ml_8x15ml", ['2', '4', '7'])
+    target_plate_loader = \
+        ("thermofisherkingfisherdeepwell_96_wellplate_2000ul", '1')
+    mastermix_labware_loader = ('nest_12_reservoir_15ml', '10')
+    sample_200ul_filtertiprack_loader = \
+        ('opentrons_96_filtertiprack_200ul', '6')
+    mastermix_300uL_tiprack_loader = ('opentrons_96_tiprack_300ul', '11')
+
+    mm_well_vol_ul = mastermix_max_vol * 1_000
+    # TODO: Remove dead volumes from the protocols - the dead-volume is
+    # already defined by the 1.5 multiplier (1/3rd of the total vol)
+    dead_vol = 1/3 * mm_well_vol_ul
+
+    mm_start_index, mm_end_index = parse_range_string(master_mix_range)
+    n_mm_wells = mm_end_index - mm_start_index + 1
+    mm_vol_per_sample = 275
+    total_mm_vol = mm_vol_per_sample * n_total_samples
+
+    # No more than 5 wells should be required for the mastermix
+    # maximum number of samples: 31*3=93, mm volume per sample = 275
+    # excess volume factor: 1.5
+    # 93*275*1.5=38,362.5 uL, max volume per mm well is 9540 uL
+    # n_wells = 38262.5/9540 = 4.02 wells
+    n_required_mm_wells = ceil(total_mm_vol/(mm_well_vol_ul-dead_vol))
+    if is_debug_mode:
+        msg = "\n\nNumber of required wells: {}\n"
+        msg = msg.format(n_required_mm_wells)
+        ctx.comment(msg)
+    if n_required_mm_wells > n_mm_wells:
+        msg = ("This protocol run requires {} wells of mastermix, but "
+               "there are only {} wells available, please ensure that there "
+               "is enough mastermix")
+        msg = msg.format(n_required_mm_wells, n_mm_wells)
+        raise Exception(msg)
+
+    # define all custom variables above here with descriptions:
+
+    # load modules
+
+    '''
+
+    Add your modules here with:
+
+    module_name = ctx.load_module('{module_loadname}', '{slot number}')
+
+    Note: if you are loading a thermocycler, you do not need to specify
+    a slot number - thermocyclers will always occupy slots 7, 8, 10, and 11.
+
+    For all other modules, you can load them on slots 1, 3, 4, 6, 7, 9, 10.
+
+    '''
+
+    # load labware
+
+    '''
+
+    Add your labware here with:
+
+    labware_name = ctx.load_labware('{loadname}', '{slot number}')
+
+    If loading labware on a module, you can load with:
+
+    labware_name = module_name.load_labware('{loadname}')
+    where module_name is defined above.
+
+    '''
+    sample_rack_1 = ctx.load_labware(
+        sample_tuberack_loader[0], sample_tuberack_loader[1][0])
+    sample_rack_2 = ctx.load_labware(
+        sample_tuberack_loader[0], sample_tuberack_loader[1][1])
+    sample_rack_3 = ctx.load_labware(
+        sample_tuberack_loader[0], sample_tuberack_loader[1][2])
+    target_plate = ctx.load_labware(
+        target_plate_loader[0], target_plate_loader[1])
```

Comment:
this is a bit extraneous... you could create a list of sample tuberacks `sample_tuberacks = [ctx.load_labware("nest_32_tuberack_8x15ml_8x15ml_8x15ml_8x15ml", s) for s in [2, 4, 7]]`

Reply:
I followed your example and loaded the tuberacks with list comprehension.